build/helper/metadata_add_all.py

```diff
@@ -237,6 +237,33 @@ def test_add_all_metadata_simple():
                 'description': 'Performs a foo, and performs it well.',
             },
         },
+        'private_method': {
```

Comment:
This corresponds to the function name as found in the C API. While functionally it makes no difference, I think you should follow the conventions for naming C functions in MI APIs.